android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/details/QuestPatientDetailActivity.kt

```diff
@@ -138,19 +142,35 @@ class QuestPatientDetailActivity : BaseMultiLanguageActivity() {
   private fun onTestResultItemClickListener(questionnaireResponse: QuestionnaireResponse?) {
     if (questionnaireResponse != null) {
       if (questionnaireResponse.questionnaire != null) {
-        val questionnaireId = questionnaireResponse.questionnaire.split("/")[1]
-        val populationResources = ArrayList<Resource>().apply { add(questionnaireResponse) }
-        startActivity(
-          Intent(this, QuestionnaireActivity::class.java)
-            .putExtras(
-              QuestionnaireActivity.intentArgs(
-                clientIdentifier = patientId,
-                formName = questionnaireId,
-                readOnly = true,
-                populationResources = populationResources
-              )
+        // TODO https://github.com/opensrp/fhircore/issues/778
+        //  1- handle via config/menu, 2- pass encounterId from quest details screen
+        if (configurationRegistry.appId == "g6pd") {
```

Comment:
k, so 778 is going to remove this app specific check? can you add a quick not about that in #778, it's clear from reading that issue that replacing this with something generic is required for it

Reply:
the above implementation has been removed here https://github.com/opensrp/fhircore/blob/70f47f52518ae4008c5681738204568e31e24819/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/details/QuestPatientDetailActivity.kt#L159-L161

Reply:
cool, is that on a branch?

Reply:
done

Reply:
yes